wasm/example/c++_get_started/build.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:io';
```

Comment:
maybe `cpp_` – just keep "weird" chars out of directories that may play weird

Reply:
I've renamed the whole c++_get_started directory to cpp_get_started.